src/DynamoSandbox/Program.cs

```diff
@@ -19,14 +22,17 @@ internal class Program
     {
         private static void MakeStandaloneAndRun(string commandFilePath, out DynamoViewModel viewModel)
         {
-            DynamoPathManager.Instance.InitializeCore(
-                Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
+            var exePath = Assembly.GetExecutingAssembly().Location;
+            var rootFolder = Path.GetDirectoryName(exePath);
+            DynamoPathManager.Instance.InitializeCore(rootFolder);
 
-            DynamoPathManager.PreloadAsmLibraries(DynamoPathManager.Instance);
+            var geometryFactoryPath = string.Empty;
+            PreloadShapeManager(ref geometryFactoryPath);
```

Comment:
The responsibilities of preloading *Autodesk Shape Manager* now falls on the client (`DynamoSandbox.exe` in this case).